sanitiser/_boundary_country.js

```diff
@@ -0,0 +1,42 @@
+var check = require('check-types');
+var iso3166 = require('iso3166-1');
+
+function sanitize(raw, clean) {
+  // error & warning messages
+  var messages = { errors: [], warnings: [] };
+
+  // init clean.boundary (if not already init)
+  clean.boundary = clean.boundary || {};
+
+  if (check.assigned(raw['boundary.country'])) {
+    var country = raw['boundary.country'];
+
+    if (!check.string(country)) {
+      messages.warnings.push('boundary.country is not a string');
+      delete clean.boundary.country;
+    }
+    else if (!containsIsoCode(country.toUpperCase())) {
+      messages.warnings.push(country + ' is not a valid ISO2/ISO3 country code');
+      delete clean.boundary.country;
+    }
+    else {
+      // the only way for boundary.country to be assigned is if input is
+      //  a string and a known ISO2 or ISO3
+      clean.boundary.country = iso3166.to3(country.toUpperCase());
+    }
+
+  } else {
+    delete clean.boundary.country;
+  }
+
+  return messages;
+
+}
+
+function containsIsoCode(isoCode) {
+  return iso3166.list().some(function(row) {
+      return row.alpha2 === isoCode || row.alpha3 === isoCode;
+  });
+}
+
+module.exports = sanitize;
```

Comment:
should this be a fatal `error` or simply a `warning`? I guess in this case the query can still execute successfully despite an input error, but it will not produce the output the developer expected.

one one hand we could follow the idea of [fail early, fail loud](https://en.wikipedia.org/wiki/Fail-fast) and on the other we can offer [fault tolerance](https://en.wikipedia.org/wiki/Fault_tolerance).

as an engineer I would choose the former but as a user I would expect the latter?

more of a question for @pelias/owners on what constitutes a failure?